### DISCUSSION TITLE
pkg show opencv -a and it display 4 different version

### DISCUSSION THREAD

**Discussion by SamYokai** [termux/termux-packages#8445] [Q&A] [answered]
Want to ask, when I run pkg show opencv -a, and it had show me 4 different versions of the opencv which are opencv-4.5.5, opencv-4.5.2, opencv-4.5.0-1, opencv-4.5.0. I would like to ask that is this many version of opencv will affect me to import the opencv into pip list? Or means did it will effect the opencv to import in Python? Since I had pkg install opencv through its-pointless repo but it doesn't get in python pip list, throws error no module named cv2 when I import cv2 in Python.

**Comment by SamYokai** [comment]
by the way, I would like to ask why there are haven't cv2 package in my /lib/python3.9/site-packages/ , although I had pkg install opencv successfully. I think I had install it properly through its-pointless-repo since scipy and numpy are no problem for me, just the opencv

**Comment by ghost** [comment, marked as answer]
`pkg show -a <pkg name>` will show all available versions for installation. The currently available version is only 4.5.5 at packages.termux.org. Others are either from third-party repos or out-of-sync mirrors. By default package manager installs only the latest available package.

This won't affect `pip list`. However our opencv may not have python bindings compiled, so importing it in Python may not work.

**Reply by SamYokai** [reply]
owh, I see. Then how can I import opencv into Python on Termux? Since from research I did a lot of people can use opencv in Python Termux. Is it need through other method to install opencv? Because I had try with package manager and also cmake also doesn't work for me. Thus, I would like to confirm is it my python version or android include Termux version will effect it.
